### DISCUSSION TITLE
v3.3

### DISCUSSION THREAD

**Discussion by ben-rogerson** [ben-rogerson/twin.macro#792] [Releases]
🎉  This version adds support for [tailwindcss@3.3.1](https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.3.1) - enjoy!

Read the [official tailwind release notes](https://github.com/tailwindlabs/tailwindcss/releases) for info on all the new goodies.

### Fixed
- Error: "Could not resolve "tailwindcss/stubs/defaultConfig.stub" https://github.com/ben-rogerson/twin.macro/issues/791

<hr /><em>This discussion was created from the release <a href='https://github.com/ben-rogerson/twin.macro/releases/tag/3.3.0'>v3.3</a>.</em>

**Comment by albohlabs** [comment]
Thank you for your awesome work. 

Tailwind now supports writing the config file in ESM or TS. [1] Is it possible to get this working with twin.macro/babel?

[1] https://tailwindcss.com/blog/tailwindcss-v3-3#esm-and-typescript-support

**Reply by ben-rogerson** [reply]
It should be possible to add this feature - twin would require some adjustments to the config loader to transpile the esm/ts formats. Feel free to submit a PR or I'll jump on it when I get some time. 

- Here’s the [tailwindcss transpilation code](https://github.com/tailwindlabs/tailwindcss/blob/cc1eec68720e168fca611956aac527575e6dd5ec/src/lib/load-config.ts#L10)
- The [twin.macro config resolver](https://github.com/ben-rogerson/twin.macro/blob/4eb928ba6c88205b70f949d606b952a9a1be2fb9/src/core/lib/configHelpers.ts) is where twin would need some adjustments

**Reply by albohlabs** [reply]
Pls have a look at https://github.com/ben-rogerson/twin.macro/pull/794 and help me out to finalize the PR.
